### ISSUE TITLE
Can't make migrations after config on django1.9 python3.4

### ISSUE THREAD

**Issue by sa-tony** [omab/python-social-auth#801]
This is the output info

``` Python
python manage.py makemigrations
/usr/local/Cellar/python3/3.4.3_2/Frameworks/Python.framework/Versions/3.4/lib/python3.4/site-packages/django/db/models/fields/subclassing.py:22: RemovedInDjango110Warning: SubfieldBase has been deprecated. Use Field.from_db_value instead.
  RemovedInDjango110Warning)
```

**Comment by hellvix** [comment]
Hummmm. So this problem comes from Social Auth. Good to know.

**Comment by tkajtoch** [comment]
This is just a django warning message. The migrations will be applied.

**Comment by tonyrft** [comment]
Yea.... Sorry for my wrong information, but please fix these warnings

**Comment by blueyed** [comment]
Dupe of / fixed by #813.

**Comment by omab** [comment]
Warnings already fixed.
